numpy/core/src/multiarray/scalartypes.c.src

```diff
@@ -34,11 +34,17 @@
 
 #include "binop_override.h"
 
+#define NP_BOOL_TRUE_REPR "numpy.True_"
+#define NP_BOOL_FALSE_REPR "numpy.False_"
```

Comment:
These defines feel like overkill to me, I'd just use the string literals directly